internals/generators/organization/config.yml.hbs

```diff
@@ -67,297 +64,133 @@ client:
         alternate_text: "{{name}}"
 {{/if}}
       links:
-        - text:
-            en: "Sign In" #link text in english
+        - text: "Sign In" #link text in english
```

Comment:
mmm...

What if a user wants to add a new link and translate it in multiple languages?
Can they do it somehow?

Reply:
Used previous getText